container/templates/ansible-container-inventory.py

```diff
@@ -4,10 +4,13 @@
 import json
 import sys
 import argparse
+import re
 
 def config_keys():
     ifs = open('/ansible-container/ansible/container.yml', 'r')
-    config = yaml.safe_load(ifs)
+    config = ifs.read()
+    config = re.sub(r"}}(?!\"|\')", "}}'", re.sub(r"(?<!\"|\'){{", "'{{", config, flags=re.M), flags=re.M)
```

Comment:
Plz document regex. It seems we probably want to go ahead and re-load/process config, no?